[2024-09-15T06:51:46.904+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.save_city_info_weather manual__2024-09-15T06:51:45.728132+00:00 [queued]>
[2024-09-15T06:51:46.911+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.save_city_info_weather manual__2024-09-15T06:51:45.728132+00:00 [queued]>
[2024-09-15T06:51:46.912+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-15T06:51:46.912+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2024-09-15T06:51:46.912+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-15T06:51:46.920+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): save_city_info_weather> on 2024-09-15 06:51:45.728132+00:00
[2024-09-15T06:51:46.926+0000] {standard_task_runner.py:55} INFO - Started process 509 to run task
[2024-09-15T06:51:46.929+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_city_info_weather_v01', 'save_city_info_weather', 'manual__2024-09-15T06:51:45.728132+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/saver_info.py', '--cfg-path', '/tmp/tmpvsziiuse']
[2024-09-15T06:51:46.930+0000] {standard_task_runner.py:83} INFO - Job 312: Subtask save_city_info_weather
[2024-09-15T06:51:46.983+0000] {task_command.py:389} INFO - Running <TaskInstance: save_city_info_weather_v01.save_city_info_weather manual__2024-09-15T06:51:45.728132+00:00 [running]> on host 0c5fe11d0c9c
[2024-09-15T06:51:47.039+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=save_city_info_weather_v01
AIRFLOW_CTX_TASK_ID=save_city_info_weather
AIRFLOW_CTX_EXECUTION_DATE=2024-09-15T06:51:45.728132+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-15T06:51:45.728132+00:00
[2024-09-15T06:51:47.041+0000] {sql.py:253} INFO - Executing: 
            create table if not exists weathers (
                id serial PRIMARY KEY ,
                city_name VARCHAR,
                max_temp VARCHAR,
                min_temp VARCHAR,
                wind VARCHAR,
                descript VARCHAR
            )
        
[2024-09-15T06:51:47.054+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-15T06:51:47.171+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-15T06:51:47.178+0000] {sql.py:364} INFO - Running statement: 
            create table if not exists weathers (
                id serial PRIMARY KEY ,
                city_name VARCHAR,
                max_temp VARCHAR,
                min_temp VARCHAR,
                wind VARCHAR,
                descript VARCHAR
            )
        , parameters: None
[2024-09-15T06:51:47.200+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=save_city_info_weather_v01, task_id=save_city_info_weather, execution_date=20240915T065145, start_date=20240915T065146, end_date=20240915T065147
[2024-09-15T06:51:47.229+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-09-15T06:51:47.251+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
